### ISSUE TITLE
C:\WINDOWS\system32\wwwroot\

### ISSUE THREAD

**Issue by flyabout** [RatScanner/RatScanner#151]
**Error**
C:\WINDOWS\system32\wwwroot\
```
System.IO.DirectoryNotFoundException: C:\WINDOWS\system32\wwwroot\
   at Microsoft.Extensions.FileProviders.PhysicalFileProvider..ctor(String root, ExclusionFilters filters)
   at Microsoft.AspNetCore.Components.WebView.Wpf.BlazorWebView.StartWebViewCoreIfPossible()
   at Microsoft.AspNetCore.Components.WebView.Wpf.BlazorWebView.OnApplyTemplate()
   at System.Windows.FrameworkElement.ApplyTemplate()
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Controls.Grid.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at MS.Internal.Helper.MeasureElementWithSingleChild(UIElement element, Size constraint)
   at System.Windows.Controls.ContentPresenter.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Wind
```
<details>
<summary>Log</summary>

```
[15:39:49.9132987] > [Error] C:\WINDOWS\system32\wwwroot\
 -------------------- 
 System.IO.DirectoryNotFoundException: C:\WINDOWS\system32\wwwroot\
   at Microsoft.Extensions.FileProviders.PhysicalFileProvider..ctor(String root, ExclusionFilters filters)
   at Microsoft.AspNetCore.Components.WebView.Wpf.BlazorWebView.StartWebViewCoreIfPossible()
   at Microsoft.AspNetCore.Components.WebView.Wpf.BlazorWebView.OnApplyTemplate()
   at System.Windows.FrameworkElement.ApplyTemplate()
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Controls.Grid.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at MS.Internal.Helper.MeasureElementWithSingleChild(UIElement element, Size constraint)
   at System.Windows.Controls.ContentPresenter.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Controls.Decorator.MeasureOverride(Size constraint)
   at System.Windows.Documents.AdornerDecorator.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Controls.Border.MeasureOverride(Size constraint)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Window.MeasureOverrideHelper(Size constraint)
   at System.Windows.Window.MeasureOverride(Size availableSize)
   at System.Windows.FrameworkElement.MeasureCore(Size availableSize)
   at System.Windows.UIElement.Measure(Size availableSize)
   at System.Windows.Interop.HwndSource.SetLayoutSize()
   at System.Windows.Interop.HwndSource.set_RootVisualInternal(Visual value)
   at System.Windows.Interop.HwndSource.set_RootVisual(Visual value)
   at System.Windows.Window.SetRootVisual()
   at System.Windows.Window.SetRootVisualAndUpdateSTC()
   at System.Windows.Window.SetupInitialState(Double requestedTop, Double requestedLeft, Double requestedWidth, Double requestedHeight)
   at System.Windows.Window.CreateSourceWindow(Boolean duringShow)
   at System.Windows.Window.CreateSourceWindowDuringShow()
   at System.Windows.Window.SafeCreateWindowDuringShow()
   at System.Windows.Window.ShowHelper(Object booleanBox)
   at System.Windows.Window.Show()
   at RatScanner.View.BlazorUI..ctor()
   at RatScanner.PageSwitcher..ctor()
[15:48:40.6954359] > [Error] C:\WINDOWS\system32\wwwroot\
 -------------------- 
 System.IO.DirectoryNotFoundException: C:\WINDOWS\system32\wwwroot\
   at Microsoft.Extensions.FileProviders.PhysicalFileProvider..ctor(String root, ExclusionFilters filters)
   at Microsoft.AspNetCore.Components.WebView.Wpf.BlazorWebVi
```
</details>

**Comment by Blightbuster** [comment]
Where did you put ratscanner at? For some reason it seems to be living in your System32?

**Comment by flyabout** [comment]
In my downloads folder, issue seems to have resolved its self somehow, no clue what happened.

**Comment by flyabout** [comment]
Ahhh, that would be it. I was starting the program using the start menu, it worked correctly when i opened in from its installation folder.

**Comment by Blightbuster** [comment]
Fixed in [v3.1.0](https://github.com/RatScanner/RatScanner/releases/tag/v3.1.0).
Please comment if this issue persists and I will reopen it.
